server/views/review/includes/additionalConditions.pug

```diff
@@ -26,7 +26,10 @@ mixin additionalConditions(editable)
             if item.text
               | !{item.text}
             if item.variable
-              span.bold #{item.variable}
+              if (Array.isArray(item.variable))
+                span.bold #{item.variable.join(', ')}
+              if (typeof(item.variable) === 'string')
+                span.bold #{item.variable}
             if item.error
```

Comment:
This kind of logic shouldn't be necessary here.

Reply:
i'll remove it and see what it does